lms/services/document_url.py

```diff
@@ -92,13 +92,7 @@ def _from_assignment_in_db(self, context, _request):
             "resource_link_id_history",  # A Blackboard course we can copy
             "ext_d2l_resource_link_id_history",  # Ditto for Brightspace
         ):
-            # Horrible work around
-            if param == "resource_link_id":
-                resource_link_id = context.resource_link_id
-            else:
-                resource_link_id = context.lti_params.get(param)
-
-            if resource_link_id and (
+            if (resource_link_id := context.lti_params.get(param)) and (
```

Comment:
We need to make a grepeable PR replacing all contex.lti_params by request.lti_params